scripts/docker-build-ext2

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+##==============================================================================
+##
+## Build a docker image and write the output to an EXT2 image
+##
+##==============================================================================
```

Comment:
Can you make this part of appbuilder?

Reply:
Yes but I would need your help with that since it requires some modification of the appbuilder options and interface. Since this script is for internal use only (and is not installed) can we merge this as a first step?